jwst/cube_build/tests/test_cube_build_step.py

```diff
@@ -0,0 +1,139 @@
+"""
+Unit test for Cube Build testing reading in MIRI cubepars ref file and using it
+"""
+
+import numpy as np
+import pytest
+from astropy.io import fits
+from jwst.datamodels import IFUImageModel
+from jwst.cube_build import CubeBuildStep
+from jwst.cube_build.file_table import ErrorNoAssignWCS
+from jwst.cube_build.cube_build import ErrorNoChannels
+from jwst.pipeline.collect_pipeline_cfgs import collect_pipeline_cfgs
+
+
+@pytest.fixture(scope='module')
+def miri_cube_pars(tmpdir_factory):
```

Comment:
Hey @jemorrison why did you create a the `cube_pars` reference file instead of using the configured file? Is there a `cube_pars` reference file?

Reply:
@mfixstsci did you mean why did I create it for this test ?
I created it for the test because I did not think it was appropriate to use CRDS in a unit test. 
So I just hacked up a dummy one (but I actually used some of the code that creates the real one)

Reply:
There is some complex logic in using the reference table and I thought I should check it in a unit test

Reply:
Hey @jemorrison, okay great. I was just curious. I noticed that you do this for all of the instruments so I figured there was a good reason for it, thanks!